src/transformers/models/whisper/modeling_flax_whisper.py

```diff
@@ -570,12 +589,12 @@ def __call__(
             else:
                 layer_outputs = decoder_layer(
                     hidden_states,
-                    attention_mask=attention_mask,
-                    encoder_hidden_states=encoder_hidden_states,
-                    encoder_attention_mask=encoder_attention_mask,
-                    init_cache=init_cache,
-                    output_attentions=output_attentions,
-                    deterministic=deterministic,
+                    attention_mask,
```

Comment:
Note to reviewer: `remat` does not support key-word arguments, hence the need to change to pure arguments